commit ae320ae1b6735e475ea49ce08dadaf332fedcfb6
Author: Filipe Laíns <lains@archlinux.org>
Date:   Tue Nov 19 09:45:23 2019 +0000

    Revert "Do not change selection when a game is updated (Closes #2144)"
    
    This reverts commit 47ecc419498abf127ee657fbc83bc313c883cb44.
    
    Signed-off-by: Filipe Laíns <lains@archlinux.org>

diff --git a/lutris/gui/lutriswindow.py b/lutris/gui/lutriswindow.py
index 9dfe092b..fbb3865d 100644
--- a/lutris/gui/lutriswindow.py
+++ b/lutris/gui/lutriswindow.py
@@ -727,6 +727,10 @@ class LutrisWindow(Gtk.ApplicationWindow):
             self.game_store.update_game_by_id(game.id)
         except ValueError:
             self.game_store.add_game_by_id(game.id)
+
+        self.view.set_selected_game(game.id)
+        if game.is_installed:
+            self.game_selection_changed(None, game)
         return True
 
     def on_search_games_fire(self, value):
diff --git a/lutris/gui/views/grid.py b/lutris/gui/views/grid.py
index 7ea802cb..3cec9c19 100644
--- a/lutris/gui/views/grid.py
+++ b/lutris/gui/views/grid.py
@@ -1,4 +1,3 @@
-"""Grid view for the main window"""
 # pylint: disable=no-member
 from gi.repository import Gtk
 from lutris.gui.views.base import GameView
@@ -45,7 +44,6 @@ class GameGridView(Gtk.IconView, GameView):
         return self.get_model().get_iter(self.current_path)
 
     def set_selected_game(self, game_id):
-        """Select a game referenced by its ID in the view"""
         row = self.game_store.get_row_by_id(game_id, filtered=True)
         if row:
             self.select_path(row.path)
